File: program/print-multiplication-table/PrintMultiplicationTable.rb
# Multiplication table in ruby...

puts "Enter the number whose multiples must be shown : "
user_input = gets
def printMultiplicationTable number
    numArray = (1..10).to_a
    numArray.each do |num|
        puts "#{number} x #{num} = #{num*number}"
    end

end
printMultiplicationTable Integer (user_input)

File: program/print-multiplication-table/PrintMultiplicationTable.cs
using System;  
public class Multiplication  
{  
    public static void Main() 
{
   int j,n;
   Console.Write("Enter a number to generate multiplication table: ");
   n= Convert.ToInt32(Console.ReadLine());   
   Console.Write("\n");
   for(j=1;j<=10;j++)
   {
     Console.Write("{0} X {1} = {2} \n",n,j,n*j);
   }
  }
}
